src/Html/Builder.php

```diff
@@ -510,6 +510,24 @@ public function addIndex(array $attributes = [])
 
         return $this;
     }
+    
+    /**
+     * Reorderable
+     *
+     * @param       $column
+     * @param array $options
+     *
+     * @return $this
+     */
+    public function reorderable($column, array $options = [])
+    {
+        $this->attributes['rowReorder'] = array_merge([
+            'dataSrc' => $column,
+            'selector' => 'tr'
```

Comment:
Is the selector really fixed to `tr`? maybe we can also set this as a parameter just in case?

```php
public function reorderable($column, $selector = 'tr', array $options = [])
```